xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/job/question/XClassBreakingQuestion.form.vm

```diff
@@ -17,15 +17,19 @@
 ## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 ## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 ## ---------------------------------------------------------------------------
+##!source.syntax=xwiki/2.1
+{{velocity}}
 #template('job/question/macros.vm')
 
```

Comment:
Those empty lines are going to cause actual, visual, new lines in the final HTML, which is not what you want.

Reply:
Thank you for your help on this. Mostly because of the dangerouseness you pointed out, I decided to revert my changes on both these files and go with the alternative solution of upgrading the velocimacro.
See more at https://github.com/xwiki/xwiki-platform/pull/3555#discussion_r1876166543

$${\color{lightgreen}ADDRESSED}$$